content/presentations.md

```diff
@@ -19,7 +19,8 @@ This presentation covers:
 - Advice and support from Teacher Training Advisers
 
 <a href="/assets/documents/train-to-teach-and-support.pdf" class="call-to-action-button">
-    Train to Teach and Support <span>slides</span>
+    <i class="fas fa-file-pdf"></i>
+    Train to Teach and Support slides
```

Comment:
Looks like the icon should be after the text here

Reply:
Yeh - this was just playing around - pushed by mistake